### ISSUE TITLE
Packaging for OpenSUSE requires chkconfig since 10.0.4

### ISSUE THREAD

**Issue by RuiSantos76** [grafana/grafana#73915]
### What happened?

RPM package for OpenSUSE requires chkconfig as a dependency, however chkconfig is provided under different name.

~ # rpm -Uhv grafana-enterprise-10.0.4-1.x86_64.rpm --test
error: Failed dependencies:
        chkconfig is needed by grafana-enterprise-v10.0.4-1.x86_64

~ # rpm -qp grafana-enterprise-10.0.4-1.x86_64.rpm --requires | grep chkconfig
chkconfig

This issue is also present on Grafana version 10.1.0 package for OpenSUSE

### What did you expect to happen?

Package should be allowed to install as the dependency is already met, but under a different name with package aaa_base

~ # zypper search --provides --match-exact chkconfig
Loading repository data...
Reading installed packages...
No matching items found.

~ # zypper search --provides --match-exact /sbin/chkconfig
Loading repository data...
Reading installed packages...

S | Name     | Summary               | Type
--+----------+-----------------------+--------
i | aaa_base | openSUSE Base Package | package

I believe the requirement for OpenSUSE, at least Leap 15.5, should be "/sbin/chkconfig"
Let me know if you require help on testing it.

### Did this work before?

Up to version 10.0.3, there was no such requirement for chkconfig:

~ # rpm -qp grafana-enterprise-10.0.3-1.x86_64.rpm --requires
/bin/sh
/bin/sh
/sbin/service
fontconfig
freetype
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
urw-fonts

### How do we reproduce it?

1. Try to upgrade or install grafana 10.0.4 or 10.1.0
2. rpm -Uhv grafana-enterprise-10.0.4-1.x86_64.rpm
3. The following message will appear: error: Failed dependencies:
chkconfig is needed by grafana-enterprise-v10.0.4-1.x86_64

### Environment (with versions)?

Grafana:  10.0.4
OS: OpenSUSE Leap 15.5
Browser: Not applicable

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by kminehart** [comment]
Thanks for the report. Much apologies, this will be resolved in the next patch release (some time this week).

**Comment by kminehart** [comment]
This should be resolved with the 10.1.1 package.


```
wget https://dl.grafana.com/oss/release/grafana-10.1.1-1.x86_64.rpm && rpm -qpR grafana-10.1.1-1.x86_64.rpm
warning: grafana-10.1.1-1.x86_64.rpm: Header V4 RSA/SHA512 Signature, key ID 10458545: NOKEY
/bin/sh
/bin/sh
/sbin/service
fontconfig
freetype
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
urw-fonts
```

It's become much easier to update our package configs recently (despite the recent hiccups). Please let me know if you continue to have issues.

**Comment by RuiSantos76** [comment]
Perfect for 10.1.1 release. Thank you for your help on this matter :smiley:
